pages/windows/scoop-bucket.md

```diff
@@ -0,0 +1,25 @@
+# scoop bucket
+
+> A bucket is an application repository.
```

Comment:
Based on the wiki page:
```suggestion
> A bucket is a Git repository with a manifest which describe how to install an application.
```
too verbose?

Reply:
This also a good idea, but I would put it a little bit more general. Maybe the term `manifest` could confuse new users. 
```
A bucket is Git repository containing files which describe how scoop installs applications.
```